packages/z/ZzFX.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "ZzFX",
+  "description": "A Tiny JS Sound FX System - Zuper Zmall Zound Zynth",
+  "filename": "ZzFX.micro.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/KilledByAPixel/ZzFX.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "sound",
+    "sounds",
+    "sound effect",
+    "sound effect engine",
+    "sound system",
+    "music",
+    "musical note",
+    "play sounds"
+  ],
+  "authors": [
+    {
+      "name": "Frank Force",
+      "email": "frankforce@gmail.com"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/KilledByAPixel/ZzFX.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "ZzFX.micro.js",
+          "ZzFX.js"
```

```suggestion
          "ZzFX.*.js"
```
Let's just include all versions

---

Done, thanks for your correction!